scripts/gcp_setup/environment/gcloud_env.py

```diff
@@ -159,6 +162,7 @@ def run_setup(self):
         self.get_organization()
         self.check_billing_enabled()
         self.has_permissions()
+        self.get_host_project()
```

Comment:
This method name might be better as ```check_network_host_project_id()```.  I know that other methods here also uses ```get``` naming, but I opened issue #842 to address

Reply:
Complete